File: src/main/java/com/train/Tester.java
package com.train;

import java.util.Scanner;

public class Tester {

    public static void main(String[] args) {
        System.out.print("Please enter number of tickets: ");
        Scanner scanner=new Scanner(System.in);
        int TotalTicket = scanner.nextInt();
        System.out.print("How many round-trip tickets: ");
        int RoundTripTicket =scanner.nextInt();
        Tickets tickets=new Tickets(TotalTicket,RoundTripTicket);
        tickets.conter();
    }
}

File: src/main/kotlin/com/KolinTrain/Tester.kt
package com.KolinTrain

import java.util.*

fun main() {
    val scanner = Scanner(System.`in`)
    println("Please enter number of tickets: ")
    var TotalTickets=scanner.nextInt()
    println("How many round-trip tickets: ")
    var RoundTripTickets=scanner.nextInt()
    val tickets=Tickets(TotalTickets,RoundTripTickets)
    tickets.counter()

}
